.github/workflows/main.yml

```diff
@@ -20,6 +20,32 @@ jobs:
           python-version: 3.10.2
       - uses: pre-commit/action@v2.0.3
 
+  test-python:
+    runs-on: ubuntu-latest
+    container: pyodide/pyodide-env:20220629-py310-chrome102-firefox100
```

Comment:
Come to think of it, does `test-python` need to be run in the docker image? We don't need browser drivers for this test.